datahub-web-react/src/app/entity/shared/tabs/Dataset/AccessManagement/AccessManagement.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Table } from 'antd';
+import { useBaseEntity } from '../../../EntityContext';
+import { GetDatasetQuery, useGetExternalRolesQuery } from '../../../../../../graphql/dataset.generated';
+import { useGetMeQuery } from '../../../../../../graphql/me.generated';
+import handleExternalRoles from './AccessRolesDetail';
+import AccessManagerDescription from './AccessManagerDescription';
+
+export default function AccessManagement() {
+    const StyledTable = styled(Table)`
```

Comment:
please place this at the top of the file, similar to toher components

Reply:
done update